scripts/validate-test-results

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env perl
+
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use strict;
+use warnings;
+
+use Mojo::JSON qw(decode_json);
+use Mojo::Path;
+use Mojo::File qw(path);
+
+my @args = @ARGV;
+my @test_results_to_check = ();
+
+while(@args){
+	my $arg = shift @args;
```

Comment:
no tabs please :)
Isn't this file linted itself?

Reply:
Next change after perltidy on `os-autoinst-commons` is a generic perltidy & perl critic workflow that can be used like the git commit checker.

as per my experiments here: https://github.com/josegomezr/perl-toolkit